kythe/go/extractors/gcp/config/common.go

```diff
@@ -84,8 +100,10 @@ func zipMergeStep() *cloudbuild.BuildStep {
 			fmt.Sprintf(
 				"%s merge --output %s %s/*.kzip",
 				constants.DefaultKzipToolLocation,
-				path.Join(outputDirectory, outputFilePattern),
+				path.Join(outputDirectory, outputFileName(corpus)),
 				outputDirectory),
 		},
+		// We explicitly don't include a WaitFor here because this should
```

Comment:
Shouldn't it then wait for the penultimate steps?

Reply:
AH yes, exactly.  I was confused on the same thing while re-writing this code.

Turns out that not having the `waitFor` *is* waiting for everything (BECAUSE THAT'S NOT CONFUSING)

Default behavior is to wait for everything, so leaving this blank is intentional.  I added this comment because I was confusing myself.

But now that you point it out, I'll go add even more info this comment

Reply:
There's a megamind meme in here somewhere...